src/RoslynPad.Common.UI/ViewModels/MainViewModelBase.cs

```diff
@@ -141,7 +149,7 @@ public MainViewModelBase(IServiceProvider serviceProvider, ITelemetryProvider te
         private IEnumerable<OpenDocumentViewModel> LoadAutoSavedDocuments(string root)
         {
             return IOUtilities.EnumerateFilesRecursive(root, DocumentViewModel.GetAutoSaveName("*")).Select(x =>
-                GetOpenDocumentViewModel(DocumentViewModel.FromPath(x)));
+                GetOpenDocumentViewModel(DocumentViewModel.FromPath(x, _documentFileWatcher)));
```

Comment:
No need to pass the watcher here - these are just autosave docs and won't appear in the tree. Moreover - even if these files were renamed or deleted, we'd still like to save them with the original autosave name.